benchmark/blankenbach/case2/ReadMe.txt

```diff
@@ -0,0 +1,24 @@
+* case2a:
```

Comment:
Your choice. But I think the comments I had should be addressed whether or not this all is included in the current patch or not.